src/web/arr/trove/chart.arr

```diff
@@ -1215,7 +1243,7 @@ fun exploding-pie-chart-from-list(
   offsets.each(check-num)
   labels.each(check-string)
   default-pie-chart-series.{
-    tab: to-table3(labels, values, offsets)
+    tab: to-table4(labels, values, offsets, range(0, labels.length()))
```

Comment:
Since `to-table3` is your custom function, and you're always passing in `range(0, labels.length())`, try using [`map3_n`](https://github.com/brownplt/pyret-lang/blob/862baff55029a75b99806833dedd11291805feaf/src/arr/trove/lists.arr#L560) instead, which will produce those indices for you.